src/core_modules/capture-core/components/Pages/Enrollment/EnrollmentPage.actions.js

```diff
@@ -65,3 +66,6 @@ export const updateTeiDisplayName = (teiDisplayName: string) =>
 
 export const setEventRelationshipsData = (eventId: string, relationships: Array<Object>) =>
     actionCreator(enrollmentPageActionTypes.SET_EVENT_RELATIONSHIPS_DATA)({ eventId, relationships });
+
+export const clickLinkedRecord = (parameters: Object) =>
```

Comment:
Hey, does it make sense to use `type URL` instead of `Object`? Thanks!

Reply:
hey @simonadomnisoru, I updated based on your suggestion. thanks!